source/medicinalproductdefinition/bundle-MedicinalProductDefinition-search-params.xml

```diff
@@ -230,4 +230,114 @@
       </SearchParameter>
     </resource>
   </entry>
+  <entry>
+    <resource>
+      <SearchParameter>
+        <id value="MedicinalProductDefinition-operation-type"/>
+        <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+          <valueCode value="trial-use"/>
+        </extension>
+        <extension url="http://hl7.org/fhir/build/StructureDefinition/path">
+          <valueString value="MedicinalProductDefinition.operation.type.reference"/>
+        </extension>
+        <url value="http://hl7.org/fhir/build/SearchParameter/MedicinalProductDefinition-operation-type"/>
+        <description value="The type of manufacturing operation, as a reference to a plan or activity"/>
+        <code value="operation-type"/>
+        <type value="reference"/>
+        <expression value="MedicinalProductDefinition.operation.type.reference"/>
+        <processingMode value="normal"/>
```

Comment:
Suggest using the target element to indicate the type of reference is to PlanDefinition or ActivityDefinition
```suggestion
        <processingMode value="normal"/>
        <target value="ActivityDefinition"/>
        <target value="PlanDefinition"/>
```